drivers/ovsdriver.go

```diff
@@ -449,17 +449,17 @@ func (d *OvsDriver) AddBgpNeighbors(id string) error {
 	// Find the switch based on network type
 	sw = d.switchDb["vlan"]
 
-	return sw.AddBgpNeighbors(cfg.Hostname, cfg.As, cfg.Neighbor)
+	return sw.AddBgp(cfg.Hostname, cfg.RouterIP, cfg.As, cfg.NeighborAs, cfg.Neighbor)
 }
 
-// DeleteBgpNeighbors deletes a bgp neighbor by named identifier
-func (d *OvsDriver) DeleteBgpNeighbors(id string) error {
+// DeleteBgp deletes bgp config by named identifier
+func (d *OvsDriver) DeleteBgp(id string) error {
 	log.Infof("delete Bgp Neighbor %s \n", id)
 	//FixME: We are not maintaining oper state for Bgp
```

Comment:
Well it is not needed with the current support hence not maintained. I made a note to see if would need it in cases of bgp restartbility scenarios without restarting netplugin. Havent gotten a chance to walk through the cases.